build.xml

```diff
@@ -210,25 +323,25 @@
       deployRoot="${source} sprint ${version}"
     />
   </target>
-
-  <!-- SYNTAX: ant local2full-specified-tests -Dversion=3.6 -Dsource=dev    -DspecifiedTests=PresalesRequestHandler_Test  optional: -DcheckOnly=true -DrunAllTests=true -->
-  <target name="local2full-specified-tests">
+  
+<!-- SYNTAX: ant local2full-specified-tests -Dversion=3.6 -Dsource=qa -DspecifiedTests=TestClass1,TestClass2   optional: -DcheckOnly=true -DrunAllTests=true -->
+<target name="local2full-specified-tests">
+    <!-- backup full first -->
+    <antcall target="backup-full">
+      <param name="version" value="${version}" />
+      <param name="type" value="package" />
+    </antcall>
+    <antcall target="backup-full">
+      <param name="version" value="${version}" />
+      <param name="type" value="destructiveChanges" />
+    </antcall>
     <antcall target="copy-destructiveChanges">
       <param name="version" value="${version}" />
       <param name="source" value="${source}" />
     </antcall>
-    <sf:deploy
-      username="${cmc.full.username}"
-      password="${cmc.full.password}"
-      serverurl="${sandbox.serverurl}"
-      pollWaitMillis="20000"
-      maxPoll="5000"
-      checkOnly="${checkOnly}"
-      testLevel="RunSpecifiedTests"
-      deployRoot="${source} sprint ${version}">
-      <runTest>${specifiedTest}</runTest> <!--PresalesRequestHandler_Test-->
-    </sf:deploy>
-  </target>
+	<!-- deploy script -->
+	<scepcificTest cmcuser="${cmc.full.username}" cmcpass="${cmc.full.password}" serverurl="${sandbox.serverurl}"/>

```

Comment:
Typo "scepcificTest".  Update names of username/password attributes per earlier comments.